docs/UserGuide.md

```diff
@@ -321,6 +321,12 @@ Examples:
 
 Clears all entries from the contact list of FINDvisor.
 
+<div markdown="span" class="alert alert-danger">:exclamation: **Warning:**<br>
+FINDvisor will **not prompt for confirmation** and will result in **permanent deletion of all contacts** in FINDvisor.
+This action is **irreversible** and there will be no way to retrieve the data unless there is a backup copy of the JSON data file.<br>
+**This command should be used with caution.**
```

Comment:
Would it be better to remove this line as I feel that the warning box is sufficient to notify the user to use this command with caution

Reply:
I think this is fine? I think having an additional statement to use the command with caution will not be too big of an issue. I can remove it if the rest feels that it is too redundant
@Dethada @YuZizhen @Arixeyeion

Reply:
I feel like the line is redundant, the warning above should be sufficient.